app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -91,5 +91,34 @@
       </div>
       </fieldset>
     </div>
+    <!--advanced options-->
+    <div>
+      <a href=""
+         ng-click="showAdvancedOptions = true"
+         ng-show="!showAdvancedOptions">Use label selectors in the storage request. </a>
```

Comment:
Not sure if the trailing space after the period will be underlined. I'd prefer to take that out.

Also a nit, but we usually leave off the period in links like these. (I know I had that in my original comment.) Also in the link just below.